communitiesuk/funding-service-design-fund-store
# FS:4413: HSRA application tool text change

_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_


### Change description
- Auction-costs section removed from fund loader config for HSRA




### How to test
- Run the scripts.load_all_fund_rounds and verify if auction costs is removed from the application


### Screenshots of UI changes (if applicable)

<img width="834" alt="Screenshot 2024-05-24 at 07 04 25" src="https://github.com/communitiesuk/funding-service-design-fund-store/assets/66220499/2a9b4f7a-d4e8-4549-946c-0fb712751fd9">
